esinstall/package.json

```diff
@@ -32,12 +32,15 @@
     "@rollup/plugin-replace": "^2.3.3",
     "cjs-module-lexer": "^0.4.0",
     "es-module-lexer": "^0.3.24",
+    "esbuild": "^0.7.13",
```

Comment:
Note: it seems like `snowpack` itself is using the same version of `esbuild` in `yarn.lock`, but it probably wouldn’t hurt just for `snowpack/package.json` to use the same semver as this. It might be weird if these ever got out-of-sync.